packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -882,10 +891,10 @@ class Navigator extends StatefulWidget {
   /// {@end-tool}
   @optionalTypeArgs
   static Future<T> pushNamed<T extends Object>(
-    BuildContext context,
-    String routeName, {
-    Object arguments,
-   }) {
+      BuildContext context,
```

Comment:
Please revert these formatting changes here and below.

Reply:
Done! Sorry it took me time to get used to flutter styling.